pkg/intermodule/account.ts

```diff
@@ -13,34 +15,48 @@ import {
 import type { ID } from '../id/type.js';
 
 export class AccountModule {
+  // NOTE: This is a temporary solution to use hono client
+  // ToDo: base url should be configurable
+  private readonly client = hc<AccountModuleHandlerType>(
+    'http://localhost:3000',
+  );
+
   constructor(private readonly controller: AccountController) {}
 
   async fetchAccount(
     name: AccountName,
   ): Promise<Result.Result<Error, Account>> {
-    const res = await this.controller.getAccount(name);
+    const res = await this.client.accounts[':name'].$get({
+      param: { name: name },
```

Comment:
```suggestion
      param: { name },
```
同名のキーとバリューなので略記できるかと